Makefile

```diff
@@ -144,22 +137,14 @@ bin/golangci-lint_${GOLANGCI_VERSION}: | bin
 	GOBIN=$(PWD)/bin go install github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_VERSION}
 	mv bin/golangci-lint $@
 
-bin/kubebuilder: | bin/kubebuilder_${KUBEBUILDER_VERSION} bin
-	ln -sf kubebuilder_${KUBEBUILDER_VERSION}/kubebuilder $@
-
-bin/kube-apiserver: | bin/kubebuilder_${KUBEBUILDER_VERSION} bin
-	ln -sf kubebuilder_${KUBEBUILDER_VERSION}/kube-apiserver $@
-
-bin/etcd: | bin/kubebuilder_${KUBEBUILDER_VERSION} bin
-	ln -sf kubebuilder_${KUBEBUILDER_VERSION}/etcd $@
-
-bin/kubectl: | bin/kubebuilder_${KUBEBUILDER_VERSION} bin
-	ln -sf kubebuilder_${KUBEBUILDER_VERSION}/kubectl $@
-
-bin/kubebuilder_${KUBEBUILDER_VERSION}: | bin
-	find $(PWD)/bin -name 'kubebuilder_*' -exec rm -r {} +
-	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${KUBEBUILDER_VERSION}/kubebuilder_${KUBEBUILDER_VERSION}_${OS}_amd64.tar.gz | tar xvz -C bin
-	ln -sf kubebuilder_${KUBEBUILDER_VERSION}_${OS}_amd64/bin $@
+# Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+test: generate fmt vet manifests
+	mkdir -p ${ENVTEST_ASSETS_DIR}
```

Comment:
this directory should be added as a dependency with a matching target; see `bin`